lib/config/hse_gparams.c

```diff
@@ -26,11 +26,10 @@
 
 struct hse_gparams hse_gparams;
 
-bool HSE_NONNULL(1, 2, 3)
-    logging_destination_converter(
-        const struct param_spec *const ps,
-        const cJSON *const             node,
-        void *const                    data)
+bool HSE_NONNULL(1, 2, 3) logging_destination_converter(
```

Comment:
Our undocumented coding style is to have newlines before function names in function definitions.

Reply:
This is clang-format because of the attribute...